torch/_dynamo/variables/user_defined.py

```diff
@@ -329,14 +329,16 @@ def call_function(
             and hasattr(
                 self.value, "__exit__"
             )  # TODO(voz): These can invoke user code!
-            and check_constant_args(args, kwargs)
-            and self.value.__init__ == object.__init__
-            and len(kwargs) == 0  # TODO(ybliang): support kwargs
+            and self.value.__new__ == object.__new__
         ):
-            unwrapped_args = [x.as_python_constant() for x in args]
+            cm_obj = tx.output.side_effects.track_object_new(
+                self.source, self.value, UserDefinedObjectVariable, {}
+            )
+            cm_obj.call_method(tx, "__init__", args, kwargs)
+
             return GenericContextWrappingVariable(
-                unwrapped_args,
-                cm_obj=self.value(*unwrapped_args),
+                args,
+                cm_obj=cm_obj,
```

Comment:
Seemed a little weird to me that we're creating two VariableTracker for this context manager object. There's the UserDefinedObjectVariable and also a GenericContextWrappingVariable. @anijain2305 thoughts?

Reply:
I think they are a bit different, ```GenericContextWrappingVariable``` need to support ```enter/exit``` in the way the ```symbolic_converter``` is using, so it's different from generic user defined object.

Reply:
So, is it ok to have two VariableTrackers (`GenericContextWrappingVariable/UserDefinedObjectVariable`) to support a generic context manager?